modules/home-manager/tmux.nix

```diff
@@ -0,0 +1,33 @@
+{ config, pkgs, lib, ... }:
+let
+  cfg = config.programs.tmux.catppuccin;
+
+  plugin = with builtins;
+    with pkgs;
+    let rev = "4e48b09a76829edc7b55fbb15467cf0411f07931";
+    in pkgs.tmuxPlugins.mkTmuxPlugin {
+      pluginName = "catppuccin";
+      version = substring 0 7 rev;
```

Comment:
I assume this is a tmux thing, but can we use the full rev?

Reply:
it isn't. most packages just slim down the commit hash or use another form of shorthand like a date, plus it's also what i usually do

Reply:
Oh, I didn't realize that was the version of the package, not the version to fetch. :)